packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -823,7 +824,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       };
     }
 
-    if (items && items.length === 0) {
+    if (items && (items.length === 0 || items.length === value.length)) {
```

Comment:
```suggestion
    if (filteredItems && filteredItems.length === 0) {
```

_nit_

Reply:
I think it would work the same, and without additional condition

Reply:
Done